src/Service/ServiceExe.cs

```diff
@@ -1514,13 +1514,13 @@ public void Start(bool forceUserTaskRerun)
             // Start IPC server on port 45684. Only start after all boot tasks are complete.
             if (EnableNetworkAccess && !DisableNetworkAccess) 
             {
-                FOServiceExe.ipcHost = new IpcServiceHostBuilder(FOServiceExe.ipcSvcProvider).AddTcpEndpoint<IFactoryOrchestratorService>("tcp", IPAddress.Any, 45684)
+                FOServiceExe.ipcHost = new IpcServiceHostBuilder(FOServiceExe.ipcSvcProvider).AddTcpEndpoint<IFactoryOrchestratorService>("tcp", IPAddress.Any, 45684, new JKang.IpcServiceFramework.Tcp.TcpConcurrencyOptions(5))
```

Comment:
what's the significance of 5 here? should we track possibly offering this as a customization?

Reply:
Seemed high enough to cover any reasonable use case of FO, without being silly large. I'm worried a bit about customization overload already so i'm hesitant to make it one.

Reply:
I would consider documenting this as the current supported # of parallel connections.